### DISCUSSION TITLE
Send additional notifications - use the emails in the checked submissions and not require Email Address(es)

### DISCUSSION THREAD

**Discussion by terryupton** [solspace/craft-freeform#366] [Feature Requests]
**Describe the bug or issue you're experiencing**
When inside the submissions panel, you can check single or multiple submissions. If you do this and want to trigger an additional notification, a model pops up to choose the template AND enter the  Email Address(es).

However, I propose that the default for this should use the email addresses from the submissions that have been checked, and function more in-line with the resend notifications. I think that the Email Address(es) field should allow you to send to those selected PLUS any addresses entered into the Email Address(es) field. 

A couple of ideas/approaches for this: 
1. It could be assumed that any checked submissions will use their email address PLUS any addresses entered into the Email Address(es) field. 

Or

2. It could auto-populate the  Email Address(es) field based on the checked submissions (emails) so it is partial automated and then you add/remove from that field. 

<img width="1147" alt="173050784-2744a0a6-37c7-42e3-9478-f5aad03554c5" src="https://user-images.githubusercontent.com/1821674/173086413-0d363bf2-0278-4b1b-a3c3-239d4a09f7c3.png">

### Example Use Case
Parents register for the website and their account is created an activated.
Admin then login and configure a few additional (manual) parameters, assigning children to classes etc.
Once admin has completed this, they head over to the registration submissions and check the submissions for the accounts they have configured (1 or more) and then manually trigger the 'Account is now setup' email. 

Currently, this is clunky as they would need to either do this one at a time and copy the email address for the user. Or keep a list as they work and paste this in. It would make more sense and be way more efficient and usable if they could trigger this as described above.

Thanks.

**Comment by terryupton** [comment]
@kjmartens - I see you moved this over to a discussion, what is the typical approach from here?
Did you have any thoughts on the above...?
It perhaps is quite specific but I do feel it would be a beneficial improvement for anyone who might use this feature.
Thanks 😄

**Comment by kjmartens** [comment]
@terryupton There are a couple of considerations here. One is that the flow you describe could be something that others don't prefer either. The intended flow for this feature is that an admin can say "I think ____ should also have received a notification about this one" and then type in their email(s) and send. The other consideration is that this feature was built specifically for a client that needed it for the aforementioned scenario.

That said, there's always room for expanding and customizing features in the future, so we certainly could add in your request some time, which is why it's in the Feature Requests category now. 🙂

**Comment by fernakr** [comment]
Came here because I was expecting the same behavior as OP. Upvoting!
